### PR TITLE
ENH: Copy index from anaconda

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

**Comment by ericdill** [comment]
@mariusvniekerk code review please

**Comment by ericdill** [comment]
@Rapportus code review please

File: example-conf.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+blacklist:
```
Comment (parente) [inline example-conf.yaml]: Comments about how these fields work would help would-be users.

File: run_tests.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+import sys
+import pytest
+
+if __name__ == '__main__':
+    # show output results from every test function
+    args = ['-v']
+    # show the message output for skipped and expected failure tests
+    args.append('-rxs')
+    print('sys.argv={}'.format(sys.argv))
```
Comment (parente) [inline run_tests.py]: Left over print?

File: test/test_conda_mirror.py [hunk 3]
```diff
@@ -1,137 +1,69 @@
...
+def repodata():
+    rd = namedtuple('repodata', ['anaconda', 'condaforge'])
+    anaconda = conda_mirror.get_repodata('anaconda', 'linux-64')
+    cf = conda_mirror.get_repodata('conda-forge', 'linux-64')
+    return rd(anaconda, cf)
+
+
+def test_match(repodata):
+    repodata_info, repodata_packages = repodata.anaconda
+    matched = conda_mirror.match(repodata_packages, {'name': 'jupyter'})
```
Comment (parente) [inline test/test_conda_mirror.py]: A wildcard test case somewhere would be good too.

**Comment by parente** [comment]
Nothing else major to contribute on this.

File: example-conf.yaml (hunk 1 above)
Comment (ericdill) [inline example-conf.yaml]: Have a look at the updates to README

Comment (ericdill) [inline example-conf.yaml]: Do those changes satisfy that request?

**Comment by ericdill** [comment]
Thanks @Rapportus @parente

**Pull request merged**